linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_network_failure_action/rule.yml

```diff
@@ -14,6 +14,7 @@ description: |-
     <tt>halt</tt>. For certain systems, the need for availability
     outweighs the need to log all actions, and a different setting should be
     determined.
+    This profile configures the <i>action</i> to be {{{ sub_var_value("var_audispd_network_failure_action") }}}.
```

Comment:
```suggestion
    This profile configures the <i>ACTION</i> to be <tt>{{{ sub_var_value("var_audispd_network_failure_action") }}}</tt>.
```

Reply:
and you can `s/sub_var_value/xccdf_value/` (`sub_var_value` still works though).